test/integration2/query/select/selects.spec.js

```diff
@@ -863,6 +863,17 @@ describe('Selects', function () {
         });
       });
 
+      it('#1982 - Allow comments in db', function() {
+        return knex('accounts')
+          .comment('Integration Comment')
+          .select('first_name', 'email')
+          .limit(1)
+          .then(function(results) {
+            expect(results).to.have.length(1);
```

Comment:
no way to test the content of the result ?

Reply:
Not sure how I could do that.